src/pytorch_lightning/strategies/single_device.py

```diff
@@ -66,6 +66,7 @@ def root_device(self) -> torch.device:
         return self._root_device
 
     def model_to_device(self) -> None:
+        assert self.model, "self.model must be set before self.model.to()"
```

Comment:
```suggestion
        assert self.model is not None, "self.model must be set before self.model.to()"
```
let's be a bit more explicit here :)